### PR TITLE
Enable Brave Translate for Desktop in Nightly

### PR DESCRIPTION
**Pull request by moritzhaller**
Resolves https://github.com/brave/brave-variations/issues/308

### CODE REVIEW

**Review by atuchin-m — DISMISSED** [review]

**Review by mihaiplesa — APPROVED** [review]

**Review by linhkikuchi — DISMISSED** [review]

**Comment by kjozwiak** [comment]
Waiting on the ✅ from @moritzhaller re: running through/verifying the above before merging into `production`. @moritzhaller mentioned that there was still some back-end work that needs to be completed before testing can be done by QA.

**Review by bsclifton — APPROVED** [review]
++

**Comment by kjozwiak** [comment]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/309#issuecomment-1194562568.

**Pull request merged**
